### PR TITLE
Remove "bc-" prefix of barclamp id's

### PR DESCRIPTION
**Pull request by vwallfahrer**

### CODE REVIEW

**Comment by vuntz** [comment]
+1

**Comment by rsalevsky** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1306,7 +1306,7 @@ EOF
 
     cd /tmp
 
-    local netfile="/opt/dell/chef/data_bags/crowbar/bc-template-network.json"
+    local netfile="/opt/dell/chef/data_bags/crowbar/template-network.json"
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: :-1: this would be the wrong name on cloud4/5 so we would have to use iscloudver 6plus

**Comment by bmwiedemann** [comment]
note: You should not include merge commits in a pull-request. please use git rebase origin/master  to drop it.

**Comment by vuntz** [comment]
+1

**Comment by toabctl** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -1189,7 +1189,12 @@ EOF
 
     cd /tmp
 
-    local netfile="/opt/dell/chef/data_bags/crowbar/bc-template-network.json"
+    if iscloudver 6plus; then
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: Actually, can you change this to:
```
if iscloudver 6plus && [[ ! $cloudsource =~ ^M[1-5]$ ]];
```

**Comment by vuntz** [comment]
Moved to https://github.com/SUSE-Cloud/automation/pull/571 due to Vanessa being away today.

**Pull request closed without merging**